test/unit/test_estimator.py

```diff
@@ -0,0 +1,99 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests for estimator class."""
+
+import json
+from unittest.mock import patch, ANY
+
+from qiskit.circuit.library import RealAmplitudes
+from qiskit.quantum_info import SparsePauliOp
+
+from qiskit_ibm_runtime.utils.json import RuntimeEncoder
+from qiskit_ibm_runtime.utils.utils import _hash
+from qiskit_ibm_runtime.qiskit.primitives.utils import _circuit_key
+
+from qiskit_ibm_runtime import Estimator, Session
+from ..ibm_test_case import IBMTestCase
+
+from .mock.fake_runtime_service import FakeRuntimeService
+
+
+class TestEstimator(IBMTestCase):
+    """Class for testing the Estimator class."""
+
+    @classmethod
+    def setUpClass(cls):
+        return super().setUpClass()
```

Comment:
You don't really need to overwrite `setUpClass` if you're just calling the parent's method.

Reply:
removed it